.ci/update-ide-properties.sh

```diff
@@ -1,3 +1,7 @@
+# Install AWS CLI
```

Comment:
`echo`?

Reply:
Updated the comment before the script to `echo`